### ISSUE TITLE
[Issue]: Ignored prefixes (e.g. "a" or "the") do not adjust to the library language

### ISSUE THREAD

**Issue by ldoench** [jellyfin/jellyfin#7074]
### Please describe your bug

When sorting by title, jellyfin ignores the English words **a** or **the** at the beginning of a title. 
But I have set my library language to German  and therefore those prefixes should **not** be ignored.
<img src="https://user-images.githubusercontent.com/12457264/147820479-4ba1efc1-5d79-46a7-b257-35415e1d0f0c.png" height=300>

Furthermore articles like **Der**, **Die**, **Das** and **Ein**, **Eine** should be ignored, because those would be the German equivalents.
<img src="https://user-images.githubusercontent.com/12457264/147820680-31cf6ad4-7758-4b2e-9453-9f0eb11a5229.png" >

I would expect this to happen automatically, but a setting to set those words manually would also work (and would also add the possibility to disable this feature)

Using the "sort title" field is not an option, because then i would have to manually edit every new added movie and also all the existing ones with german prefixes.

### Jellyfin Version

10.7.7

### Environment

```markdown
- OS: Ubuntu
- Virtualization: No
- Clients: Browser
- Browser: Chrome 96
- FFmpeg Version: 4.3.1-Jellyfin
- Playback Method: n.a.
- Hardware Acceleration: n.a.
- Plugins: Trakt
- Reverse Proxy: none
- Base URL: n.a.
- Networking: n.a.
- Storage: n.a.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by WWWesten** [comment]
Quick workaround hides in system.xml
```
  <SortRemoveWords>
    <string>the</string>
    <string>a</string>
    <string>an</string>
  </SortRemoveWords>
```
Just remove unneeded strings and add own needed.
`    <string>das</string>`
and so forth.

But it's probably better to store such strings in a translation file. So when switching the interface to the language where it is important: German, all Romance, etc., it would happen automatically.

**Comment by ldoench** [comment]
I tried the workaround and edited the system.xml, but the sorting is still the same. 
I already did a rescan of the library and tried adding it as a new library, but still Der, Die, Das are not ignored. Do I have to delete a database? Would be sad, because I would like to keep the edits to the metadata, i already made.

From an UX perspective the best place for such a setting would be under library/metadata next to the country and the language.

**Comment by WWWesten** [comment]
After configuration, Jellyfin was restarted, is right? And after customization, were new items with articles at the beginning of the title added to the Jellyfin? Does sorting work for them?

**Comment by ldoench** [comment]
I stopped the service, edited the system.xml and then restarted it.
For testing purposes I added a new item, but it still gets sorted under "Der".

**Comment by WWWesten** [comment]
I has experiment some. I configured system.xml with "der" and "das". I'm added fake movies "Das Filmmm (2010)", "Der Filmmm (2010)", "Paradise Galaxy (1956)". In "Movies" screen they are ordered by 1srt word (include articles) but by alphabetical menu in right side of the screen, "Das Filmmm (2010)" and "Der Filmmm (2010)" items are nested under "F" letter and not under "D" letter. "Paradise Galaxy (1956)" nested under "P" letter. So it works for that vertical menu.

**Comment by ldoench** [comment]
<img width="710" alt="Bildschirmfoto 2022-01-02 um 18 44 09" src="https://user-images.githubusercontent.com/12457264/147884507-bcfec29a-9fa1-46d3-bd23-9c8c3ddd8c2c.png">

I selected D in the vertical menu, but still movies with articles are shown there.

**Comment by WWWesten** [comment]
No idea more... May be You will want post about it to Reddit or Matrix.

**Comment by ldoench** [comment]
Hi, I asked on Reddit and Matrix, but was not able to fix this issue

**Comment by GerLan** [comment]
Hi,
it's working if you rescan your library after you have changed the system.xml

<SortRemoveWords>
    <string>the</string>
    <string>a</string>
    <string>an</string>
    <string>der</string>
    <string>die</string>
    <string>das</string>
    <string>ein</string>
    <string>eine</string>
    <string>einer</string>
    <string>la</string>
    <string>le</string>
    <string>el</string>
  </SortRemoveWords>

**Comment by jmfergeau** [comment]
Same for me, it doesn't work. Even after soft restarting, even after hard restarting, rescanning libraries every time, but it still doesn't work in 10.10.7. Is it at least fixed in 10.11?

**Comment by CutterXYZ** [comment]
> Hi, it's working if you rescan your library after you have changed the system.xml

Thanks, this worked for me after restarting Jellyfin and doing a full library rescan (not replacing existing images).

I agree with the above comments that the setting should be language-dependent.
